ibis/expr/types/generic.py

```diff
@@ -63,20 +63,17 @@ def type(self) -> dt.DataType:
         """Return the [DataType] of this expression."""
         return self.op().output_dtype
 
-    def hash(self, how: str = "fnv") -> ir.IntegerValue:
+    def hash(self) -> ir.IntegerValue:
```

Comment:
Do we have a generic backend test that check that the output type is integral, and consistent across two executions? Something like:

```python
def test_hash(t):
    h1 = t.x.hash().execute()
    h2 = t.x.hash().execute()
    assert h1 == h2
    assert h1.dtype == "i8"  # or should this be "u8"? Do we care about resolution or only it's an integer?
```

Reply:
:+1: added and also added the op across a few more backends.